### PR TITLE
KOGITO-6197: Implement integration tests for JS/TS API in kie-editors-standalone package

### PR DESCRIPTION
**Pull request by domhanak**
WIP
cc @romartin the usage of the API I mentioned on meeting. I am writing some cypress checks now. Note that the application can be enhanced, this is just mvp :) 

@jstastny-cz Feel free to take a look, I still need to add tests and improve the Component

### CODE REVIEW

**Review by romartin — APPROVED** [review]
Thanks @domhanak , looks great!

**Comment by romartin** [comment]
Adding also @handreyrc as reviewe, he implemented these new methods on the API so he'll have more details about its usage. So I think it's just good to double check how this test does, and if it does everything as expected.
Thanks guys!

File: packages/kie-editors-standalone/it-tests/src/components/canvas/CanvasNodeComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+}
+
+export interface State {
+  currentColor: string;
+  currentBorderColor: string;
+  location: number[];
+  absoluteLocation: number[];
+}
+
+export class CanvasNodeComponent extends React.Component<Props, State> {
```
Comment (tiagobento) [inline packages/kie-editors-standalone/it-tests/src/components/canvas/CanvasNodeComponent.tsx:32]: Hi @domhanak, can you please change this component to be a function component? We're not creating new components as classes anymore.

Comment (domhanak) [reply packages/kie-editors-standalone/it-tests/src/components/canvas/CanvasNodeComponent.tsx:32]: Yea, making it as such, just redoing the whole component to be dropdown with select that updates on editor changes

**Comment by tiagobento** [comment]
@domhanak What's the status of this PR? Asking because if you're interested in testing it with the CI, I recommend you close this one and open a PR on your fork. The CI will run normally for you to verify, we won't have a stale PR on `kiegroup/kogito-tooling`.

**Comment by tiagobento** [comment]
@domhanak ^ ?

**Comment by domhanak** [comment]
Will  reopen later :)

**Pull request closed without merging**